includes/admin/admin-pages.php

```diff
@@ -447,6 +447,10 @@ function edd_is_admin_page( $passed_page = '', $passed_view = '' ) {
 			// Supported post types
 			} elseif ( edd_is_insertable_admin_page() ) {
 				$found = true;
+
+			// Mislinked EDD settings screen
+			} elseif ( 'admin.php' === $pagenow && 'edd-settings' === $page ) {
```

Comment:
I'm wondering if this part is really even needed:

```php
'admin.php' === $pagenow && 
```

If we have `?page=edd-settings` at all then I think that's a decent assumption it should be considered an EDD admin screen.

(Just thinking that being very specific is what caused this issue in the first place.)